src/components/event/EventActions.tsx

```diff
@@ -0,0 +1,88 @@
+import { IEvent, isPastEvent } from '@/model/event';
+import React, { useMemo } from 'react';
+import { AddToCalendar, defaultEventDuration } from './AddToCalendar';
+import {
+  CalendarIcon,
+  MapPinIcon,
+  TicketIcon
+} from '@heroicons/react/24/outline';
+import { DateTime } from 'luxon';
+import { SlSocialYoutube } from 'react-icons/sl';
+import Link from 'next/link';
+
+type Props = {
+  event: IEvent;
+};
+
+const EventActions: React.FC<Props> = ({ event }: Props) => {
+  const eventDate = useMemo(() => DateTime.fromISO(event.date), [event.date]);
+  const isPast = useMemo(() => isPastEvent(event), [event]);
+
+  const formattedDate = eventDate.toLocaleString(
+    DateTime.DATETIME_MED_WITH_WEEKDAY,
+    { locale: 'it' }
+  );
+
+  const EventAction = ({
```

Comment:
in order to avoid defining this component on every renderer, could it be moved outside `EventActions` as aa standalone component (we could keep it in the same file)?

Reply:
Great tip